### PR TITLE
fix: missing typegraphs

### PR DESCRIPTION
**Pull request by zifeo**
- fix the typegraphs that were incorrectly formatted

- [ ] The change comes with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

File: examples/typegraphs/microservice-orchestration.py [hunk 1]
```diff
@@ -25,17 +25,15 @@ def team_a(g: Graph):
...
-#     g.expose(
-#         public,
-#         version=deno.static(t.integer(), 12),
-#         record=deno.static(t.struct({"weight": t.integer()}), {"weight": 100}),
-#     )
+@typegraph(
+    # skip:next-line
+    cors=Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
+)
+def team_b(g: Graph):
```
Comment (destifo) [inline examples/typegraphs/microservice-orchestration.py:32]: The error on the test failure is different, but shouldn't the ts and python version of typegraph have the same name?

**Review by destifo — CHANGES_REQUESTED** [review]

**Comment by destifo** [comment]
So the missing typegraphs issue came from `stderr` of the ruff formatting command not being handled?

File: examples/typegraphs/microservice-orchestration.py (hunk 1 above)
Comment (zifeo) [reply examples/typegraphs/microservice-orchestration.py:32]: I see them having the same name, can you point at where they differ?

**Comment by zifeo** [comment]
@destifo quite unsure yet, but at least we would have the debug logs for that now - more knowledge, better idea where the failure is :)

File: examples/typegraphs/microservice-orchestration.py (hunk 1 above)
Comment (destifo) [reply examples/typegraphs/microservice-orchestration.py:32]: overlooked the second typegraph on my part, all looks good.

**Review by destifo — APPROVED** [review]
lgtm

**Pull request merged**